modules/docs/pages/replication-protocol.adoc

```diff
@@ -169,7 +169,7 @@ _(optional)_ +
 `versioning`: `rev-trees` (default) or `version-vectors` -- see the <<revIDs>> section. +
 _other properties_: Named parameters for the filter function
 _(optional)_ +
-`sendReplacementRevs`: Boolean indicating whether the passive peer (pusher) should send replacement revs rather than NoRev when the body of a requested rev is unavailable and a newer revision is available. _(optional)_+
+`sendReplacementRevs`: Boolean indicating whether the passive peer (pusher) should send replacement revs rather than NoRev when the body of a requested rev is unavailable and a newer revision is available. _(optional)_ +
```

Comment:
Random space added?

Reply:
It's not random, it is required to put this text on a newline. https://docs.asciidoctor.org/asciidoc/latest/blocks/hard-line-breaks/

Without this, the next line would be on the same line.